Automattic/newspack-block-theme
# feat: add support for jetpack top posts

### All Submissions:






### Changes proposed in this Pull Request:



This PR adds super simple support for Jetpack's Top Posts block. Nothing fancy, just correct font-sizes, colours, spacing.

### How to test the changes in this Pull Request:

1. You need to be online with Jetpack active
2. Switch to this branch
3. Add Top Posts block to a page
4. Check if it works OK (note: there' a bug in the editor in Grid mode -- editor doesn't look like the front end unless we display 3/6/9 posts -- not our fault here)

### Other information:
